app/page.tsx

```diff
@@ -82,7 +82,7 @@ export default function Home() {
         );
       }
     } else if (data.sessionDetail.platform === 'quiz') {
-      generateQuizLink(data.sessionDetail.platform_link)
+      generateQuizLink(data.sessionDetail.platform_link, userId || '')
```

Comment:
Throw error  (or maybe just ignore here) if no userID present, because without userID the quiz link won't work anyway

Reply:
done!